File: LexerPLSharpVersion/LexerPLSharpVersion/constants/MathOperators.cs
using LexerPLSharpVersion.token;

namespace LexerPLSharpVersion.constants {
   public static class MathOperators {
      private const char Plus = '+';
      private const char Minus = '-';
      private const char Multiply = '*';
      private const char Divide = '/';
      
      public static bool IsMathOperator(char character) {
         switch (character){
            case Plus:
            case Minus:
            case Divide:
            case Multiply:
               return true;
            default:
               return false;
         }
      }
      
      public static TokenType DetectMathOperatorType(char character) {
         switch (character){
            case Divide:
               return TokenType.Divide;
            case Minus:
               return TokenType.Minus;
            case Multiply:
               return TokenType.Multiply;
            case Plus:
               return TokenType.Plus;
            default:
               return TokenType.Unknown;
         }
      }
   }
}

File: PL0/src/main/java/lexer/constants/MathOperators.java
package lexer.constants;

import token.TokenType;

public class MathOperators {
    public static final char PLUS = '+';
    public static final char MINUS = '-';
    public static final char MULTIPLY = '*';
    public static final char DIVIDE = '/';

    public static boolean isMathOperator(char character) {
        switch (character){
            case PLUS:
            case MINUS:
            case DIVIDE:
            case MULTIPLY:
                return true;
            default:
                return false;
        }
    }

    public static TokenType detectMathOperatorType(char character) {
        switch (character){
            case DIVIDE:
                return TokenType.DIVIDE;
            case MINUS:
                return TokenType.MINUS;
            case MULTIPLY:
                return TokenType.MULTIPLY;
            case PLUS:
                return TokenType.PLUS;
            default:
                return TokenType.UNKNOWN;
        }
    }
}
